pkg/media/rtp/conn.go

```diff
@@ -153,13 +161,21 @@ func (c *Conn) ListenAndServe(portMin, portMax int, listenAddr string) error {
 
 func (c *Conn) readLoop() {
 	conn, buf := c.conn, c.readBuf
+	overflow := false
 	var p rtp.Packet
 	for {
 		n, srcAddr, err := conn.ReadFromUDP(buf)
 		if err != nil {
 			return
 		}
 		c.dest.Store(srcAddr)
+		if n > MTUSize {
+			if !overflow {
+				overflow = true
+				c.log.Errorw("RTP packet is larger than MTU limit", nil)
```

Comment:
I'd prefer to not pass broken packet. Go crashes are fine and easy to recover from/fix, but crashes from C code are not. We could specifically fuzz the system to (hopefully) detect those.

As for the error/warn, even though this case won't break the system, it will still cause audio glitches in a specific call. I'd like to know when it happens, especially when a fix is as easy as increasing the buffer size. So I'd like to keep it as an error here.